backend/validators/user.validator.ts

```diff
@@ -0,0 +1,56 @@
+import { Request, Response, NextFunction } from 'express';
+import { schema as socialSchema } from './social.validator';
+import Joi from 'joi';
+import StatusCode from '../utils/StatusCode';
+
+const validateBody = (schema: Joi.ObjectSchema) => {
```

Comment:
I see that we are using these methods twice and will be using for each models. Could we move these global validation to different file and import them ?

Reply:
Sample requests

![api](https://user-images.githubusercontent.com/29055880/113352784-09b3d280-933d-11eb-9878-6bb9573fe2ee.jpg)
<br/>
**GET /api/users/:id**
![postman](https://user-images.githubusercontent.com/29055880/113352786-0a4c6900-933d-11eb-8b42-e0d65ba977d7.jpg)